src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1633,27 +1633,19 @@ private RefreshItem getRefreshToken(final AuthenticationRequest request) {
             if (item == null || StringExtensions.IsNullOrBlank(item.getRefreshToken())) {
                 // if not present, check multiResource item in cache. Cache key
                 // will not include resourceId in the cache key string.
-                Logger.v(TAG, "Looking for Multi Resource Refresh token");
+                Logger.v(TAG + methodName, "Looking for Multi Resource Refresh token");
                 keyUsed = CacheKey.createMultiResourceRefreshTokenKey(request, userId);
                 item = mTokenCacheStore.getItem(keyUsed);
                 multiResource = true;
             }
             
-            // If still cannot find an item, try with family clientid
```

Comment:
Usually the best way temporary disable code is to wrap around if (isFeatureEnabled) and set it to false...unless you're going with totally different design